crates/client/rpc/src/constants.rs

```diff
@@ -2,3 +2,9 @@
 pub const MAX_EVENTS_KEYS: usize = 100;
 /// Maximum number of events that can be fetched in a single chunk for the `get_events` RPC.
 pub const MAX_EVENTS_CHUNK_SIZE: usize = 1000;
+
+/// Path to the `genesis.json` file
+pub const GENESIS_FILE: &str = "configs/genesis-assets/genesis.json";
```

Comment:
This should not be hardcoded. And it should not be treated from this path.
Use `load_genesis` fn defined in ` crates/node/src/chain_spec.rs`

Reply:
`crates/node/src/chain_spec.rs` cannot be used in `crates/client/...` because it would result in cyclic dependencies. That's why I basically copy/pasted it. Maybe it can be moved to another file however, and `use`d in both places.

Reply:
Ya I agree with moving it to a utils crate - https://github.com/keep-starknet-strange/madara/pull/1265#discussion_r1404457326